analysis/mrr/total_mrr_by_month.sql

```diff
@@ -0,0 +1,84 @@
+with
```

Comment:
When I use a model twice in a query like this, I create a CTE at the beginning of the query so that I can only reference the table a single time; subsequently I just reference that CTE. I recommend that approach to improve simplicity / modularity.